### ISSUE TITLE
GraphAligner slow on R10 reads mapped to hprc graph

### ISSUE THREAD

**Issue by xchang1** [maickrau/GraphAligner#106]
Hi @maickrau,

I'm trying to map Nanopore R10 and Hifi reads to the HPRC graph and its super slow- I'm mapping 1 million reads on 64 threads and it still hasn't finished after 20 hours. Is this what you'd expect or do you have any advice for speeding it up?

I'm mapping to the v1.1 chm13 minigraph cactus frequency filtered [hprc graph](https://s3-us-west-2.amazonaws.com/human-pangenomics/pangenomes/freeze/freeze1/minigraph-cactus/hprc-v1.1-mc-chm13/hprc-v1.1-mc-chm13.d9.gbz). I used vg to unchop the nodes to combine the smaller nodes with nothing between them and then convert it to gfa
```
vg mod -u hprc-v1.1-mc-chm13.d9.hg -t 20 > hprc-v1.1-mc-chm13.d9.unchopped.hg
vg convert -f -t 20 hprc-v1.1-mc-chm13.d9.unchopped.hg > hprc-v1.1-mc-chm13.d9.unchopped.gfa
```
This is the command I ran
```GraphAligner -t {threads} -g {input.gfa} -f {input.fastq} -x vg --multimap-score-fraction 1.0 -a {output.gam}```
I get a bunch of errors like this, but it's still running
```
src/GraphAligner.h:685: Assertion 'trace.trace[i].nodeSwitch || trace.trace[i].DPposition.node != trace.trace[i+1].DPposition.node || trace.trace[i].DPposition.nodeOffset != trace.trace[i+1].DPposition.nodeOffset' failed. Read: m64011_190714_120746/94503957/ccs. Seed: 0+,0,0,0
src/GraphAlignerBitvectorCommon.h:560: Assertion 'traceEnd >= traceStart' failed. Read: m64011_190714_120746/94634063/ccs. Seed: 0+,0,0,0
```

I appreciate any help you can give
Thanks!
Xian

**Comment by maickrau** [comment]
Yeah it seems to be pretty slow on that graph with `-x vg`. You can try these parameters instead: 
```
--seeds-mxm-length 30 --seeds-mem-count 10000 --bandwidth 15 --multimap-score-fraction 0.99 --precise-clipping 0.85 --min-alignment-score 100 --clip-ambiguous-ends 100 --overlap-incompatible-cutoff 0.15 --max-trace-count 5 --mem-index-no-wavelet-tree
```
which is slightly modified from the parameters that verkko uses for aligning to assembly graph, seems to work decently for the HPRC graph at least for ONT reads.

**Comment by xchang1** [comment]
Awesome, thanks so much!
